_posts/2025-08-04-Hosting-OpenSearch-MCP-server-with-Bedrock-AgentCore.md

```diff
@@ -0,0 +1,269 @@
+---
+layout: post
+title: "Hosting the OpenSearch MCP server with Amazon Bedrock AgentCore"
+authors:
+ - jiapingzeng
+date: 2025-08-04
+categories:
+ - technical-post
+meta_keywords: "MCP, Agentic, Agent, Model context protocol, MCP server, LLM integration, observability agent, Bedrock AgentCore, hosted MCP server"
+meta_description: "Learn how to deploy the OpenSearch MCP server with Amazon Bedrock AgentCore Runtime."
```

Comment:
Updated meta:
_meta_keywords:_ OpenSearch MCP Server, Model Context Protocol, OpenSearch clusters, Amazon Bedrock, Bedrock AgentCore, AWS CloudFormation, OpenSearch integration, AI agents

_meta_description:_ Learn to deploy OpenSearch MCP server on Bedrock AgentCore Runtime to securely connect AI agents with OpenSearch clusters using CloudFormation or CLI methods.